applications/HDF5Application/python_scripts/core/operations/system.py

```diff
@@ -0,0 +1,42 @@
+'''HDF5 operating system operations.
+
+license: HDF5Application/license.txt
+
+Main authors:
+    Philipp Bucher
```

Comment:
from my side you don' have to include me if this would make the specification of authors / license superfluous

is there a specific reason why you don't use the license from the core?